test-fixtures/urn/f09/metadata.json

```diff
@@ -0,0 +1,7 @@
+{
+  "description": "we should test generating the URN in case of no ISBN",
```

Comment:
Good test case catch

Reply:
(Just converted this from ye olde tests, it was there already)